app/services/facility_region_csv.rb

```diff
@@ -0,0 +1,47 @@
+class FacilityRegionCsv
```

Comment:
Should this go into `app/exporters` like our other CSV classes?

Reply:
Right now I think its still pretty specific to this BD purpose -- if we end up generalizing it I think it should definitely move there.